src/components/core/PivotTable.tsx

```diff
@@ -394,6 +406,18 @@ export class PivotTableInner extends
             </div>
         );
     }
+
+    private getMeasureFormat(params: any): string {
```

Comment:
There is a more defensive and simpler way:
`
const measureGroupHeader = headers.find(isMeasureGroupHeader)
invariant(measureGroupHeader, 'measureGroup missing in dimension[1]');
return header.measureGroupHeader.items[measureIndex].measureHeaderItem.format
`